CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Change Log
 
 ## [Unreleased]
-### Added
+### Added
+- Papertrail - Support PaperTrail::Version model.
```

Comment:
This is not a really meaningful change. What do you support? Reading the code, I guess:
- Changes made using Forest are now tracked with Papertrail.

But I don't get where in the code you added the feature to make the Papertrail `versions` visible in Forest.